### PR TITLE
adding visual emphasis on user-facing stuff

### PR DESCRIPTION
**Pull request by lmichael107**

### CODE REVIEW

File: covid-19.md [hunk 1]
```diff
@@ -6,29 +6,34 @@ title: Extra Support for COVID-19 Research
...
+
+OSG is helping to organize extra resources to be made available to COVID-19 research!  These resources include:
 
-- **Consultation and facilitation** for quickly scaling up and automating workflows.
-- **Expertise** on using high throughput computing for maximizing research throughput.
-- **Access to computing capacity** through the [OSG Connect](https://support.opensciencegrid.org/support/home)
-  service which provides a batch system submit environment based on HTCondor.  The OSG aggregates
+- **Proactive consultation and facilitation** for quickly scaling up and automating workloads with up to [thousands of 
+concurrently-running jobs](https://gracc.opensciencegrid.org/dashboard/db/covid-19-research?orgId=1).
+- **Expertise** on using high throughput computing for maximizing research throughput, perhaps including
```
Comment (matyasselmeci) [inline covid-19.md:17]: Not a big fan of "perhaps including development effort" -- the "perhaps" sounds wishy-washy.

File: covid-19.md [hunk 2]
```diff
@@ -6,29 +6,34 @@ title: Extra Support for COVID-19 Research
...
+- **Expertise** on using high throughput computing for maximizing research throughput, perhaps including 
+development effort.
+- **Access to computing capacity** through the *[OSG Connect](https://support.opensciencegrid.org/support/home)*
+  service which provides a batch system submit environment based on HTCondor (allowing up to millions of jobs submitted via a
+  single submit file).  The OSG aggregates
   about 100 clusters across the US and COVID-19 research could access tens of thousands of cores
   through this mechanism.  The OSG also provides a CDN for distributing data to jobs.
+- **Capabilities for a variety of computational research tasks.** See [Computation on the OSG](https://opensciencegrid.org/about/computation/).
 
+Sites Supporting COVID-19 Research
```
Comment (matyasselmeci) [inline covid-19.md:26]: Maybe this should be "Submitting to Sites Supporting COVID-19 Research"? The current headline makes it seem like I'm about to get a list of sites, and then the paragraph started talking about researchers, and I got confused.

File: covid-19.md (hunk 1 above)
Comment (lmichael107) [reply covid-19.md:17]: Hi Matt,
That's why I moved the information/link to GRACC under this heading. The information under this heading is now ALL related to sites, and information for running COVID-19 work is under the first section.

Comment (lmichael107) [reply covid-19.md:17]: At the very least, if you merge as-is, I noticed another grammatical error that will need to be fixed, too.

Comment (matyasselmeci) [reply covid-19.md:17]: OK, should I wait for you to commit a fix?

Comment (lmichael107) [reply covid-19.md:17]: no. please merge!

**Pull request merged**